{{cookiecutter.project_slug}}/.travis.yml

```diff
@@ -44,7 +44,11 @@ jobs:
         - docker pull {{cookiecutter.docker_username}}/{{cookiecutter.project_slug}}:${IMAGE_TAG}
       deploy:
         provider: script
-        script: ./scripts/push_to_hub.sh
+        script:
```

Comment:
Just a note: Travis will run all the commands here even if one of them errors. Of course, these should not be causing an error :)